events/imperia_frankfurt_parliament_events.txt

```diff
@@ -931,6 +931,12 @@ frankfurt_parliament.8 = {
 		# Get every establishment aligned country
 		every_in_global_list = {
 			variable = german_establishment_aligned
+			# Do not subjugate if the country left the Parliament
+			limit = {
+				NOT = {
+					has_variable = left_frankfurt_parliament
+				}
+			}
```

Comment:
wait multiple countries can accept the gutter crown and have this variable? xd

Reply:
Why is it not removed when leaving the parliament?

Reply:
This variable indicates that they left parliament, it is needed when the Gutter King asks for submission and now when the German major capitulates during the German civil war.

It is cleaned up in the variable cleanup.

And yes, the Gutter king can in theory leave parliament and still be voted to become the Gutter king, thats intended.